core/config/project_settings.cpp

```diff
@@ -676,7 +677,12 @@ void ProjectSettings::clear(const String &p_name) {
 }
 
 Error ProjectSettings::save() {
-	return save_custom(get_resource_path().plus_file("project.godot"));
+	print_line("save");
```

Comment:
That's going to get annoying pretty fast :)

Reply:
Oops